front/app/modules/commercial/custom_idea_statuses/admin/containers/index.tsx

```diff
@@ -70,17 +66,19 @@ const StyledIconTooltip = styled(IconTooltip)`
 `;
 
 const IdeaStatuses = () => {
-  const ideaStatuses = useIdeaStatuses();
+  const { data: ideaStatuses, isLoading } = useIdeaStatuses();
```

Comment:
We now have an easy access to `isLoading` helper that we can use to show the appropriate ui state (a spinner in this case).